dlpy/model.py

```diff
@@ -2012,83 +2029,78 @@ def display(self, layer_id, filter_id=None):
         plt.show()
 
 
-class DLPyDict(collections.MutableMapping):
```

Comment:
also why removing this?

Reply:
I think the class is better to move to utils.py